changelog.d/6996.bugfix

```diff
@@ -0,0 +1 @@
+Properly handle the result of FederationClient.get_pdu as a coroutine.
```

Comment:
could you try to give more info as to the symptoms this is fixing?

Reply:
Updated this in 9d8e306 after a discussion on Matrix.